docs/Webhook-Parameters.md

```diff
@@ -14,7 +14,7 @@ Usage of webhook:
   -hotreload
         watch hooks file for changes and reload them automatically
   -http-methods string
-        globally restrict allowed HTTP methods; separate methods with comma
+        set default allowed HTTP methods (ie. "POST"); separate methods with comma
```

Comment:
This isn't something I've changed, just something where the snapshot of the `-help` message in this file had drifted out of date with respect to the code.